sinon/index.d.ts

```diff
@@ -109,6 +109,7 @@ declare namespace Sinon {
         callsArgOnAsync(index: number, context: any): SinonStub;
         callsArgWithAsync(index: number, ...args: any[]): SinonStub;
         callsArgOnWithAsync(index: number, context: any, ...args: any[]): SinonStub;
+        callsFake(...args: any[]): SinonStub;
```

Comment:
The argument is a callback function. The deprecated method had `func: any`. I'll change it for `func: (...args: any[]) => void`